src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs

```diff
@@ -207,6 +225,35 @@ static async ValueTask<QuicConnection> StartConnectAsync(QuicClientConnectionOpt
     /// </summary>
     public IPEndPoint LocalEndPoint => _localEndPoint;
 
+    private async void OnStreamCapacityIncreased(int bidirectionalIncrement, int unidirectionalIncrement)
+    {
+        // Bail out early to avoid queueing work on the thread pool as well as event args instantiation.
+        if (_streamCapacityCallback is null)
+        {
+            return;
+        }
+
+        // Do not invoke user-defined event handler code on MsQuic thread.
+        await Task.CompletedTask.ConfigureAwait(ConfigureAwaitOptions.ForceYielding);
```

Comment:
How is that different from `await Task.Yield();`?

Reply:
It's effectively `await Task.Yield().ConfigureAwait(false)`